planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -808,6 +820,41 @@ void planning_scene_monitor::PlanningSceneMonitor::currentWorldObjectUpdateCallb
     }
 }
 
+void planning_scene_monitor::PlanningSceneMonitor::syncSceneUpdates(const ros::Time &t)
+{
+  if (t.isZero())
+    return;
+
+  ROS_DEBUG_STREAM_NAMED("PSM", "sync to: " << fmod(t.toSec(), 10.));
+  enforce_next_state_update_ = true;  // enforce next state update to trigger without throttling
```

Comment:
@rhaschke Could you please elaborate a bit on the use of that flag-variable?
Afaics, the way this is currently used it only enforces *one* unthrottled update of the robot state, that is triggered by the next joint state message that arrives.
However, if that message is still not the one we are waiting for, the next updates are throttled again, is that correct?
If it is, then why?